plan.go

```diff
@@ -111,6 +112,8 @@ func processWork(ctx context.Context, w work) {
 	o := output{builder: w.builder}
 	fn := reflect.ValueOf(w.builder.Builder)
 	args := make([]reflect.Value, 1)
+	trace, ctx := tracing.NewInternalSpan(ctx, w.builder.Name)
+	defer trace.End()
```

Comment:
worth capturing the error from the function call?

```Go
o.outputs = fn.Call(args)
if !o.outputs[1].IsNil() {
    trace.SetError(o.outputs[1].Interface().(error))
}
```

Reply:
sure, let me update

Reply:
done